packages/expo-web-browser/ios/WebBrowserSession.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+import SafariServices
+import ExpoModulesCore
+
+internal class WebBrowserSession: NSObject, SFSafariViewControllerDelegate {
+  let viewController: SFSafariViewController
+  var modalNavigationController: UINavigationController?
```

Comment:
This seems unneeded as you're only referencing in a single place in the code by `modalNavigationController = nil`
```suggestion
```